src/metabase/cmd.clj

```diff
@@ -168,53 +152,43 @@
 
   `--mode` can be one of `:update` or `:skip` (default). `--on-error` can be `:abort` or `:continue` (default)."
   [path & options]
-  (apply import* path false options))
+  (log/warn (u/colorize :red (trs "''load'' is deprecated and will be removed in a future release. Please migrate to ''import''.")))
+  (let [opts (merge {:mode     :skip
+                     :on-error :continue}
+                    (m/map-vals mbql.u/normalize-token (cmd-args->map options)))]
+    (call-enterprise 'metabase-enterprise.serialization.cmd/v1-load path opts)))
```

Comment:
This `merge` with the default options (which if the v1 and v2 functions actually _do_ take the same default options, could be a single `def` but for now, they seem to diverge) and a call to v1-load replaces the complex and limited vararg-based default logic.